src/domain/cohort.js

```diff
@@ -4,12 +4,20 @@ import dbClient from '../utils/dbClient.js'
  * Create a new Cohort in the database
  * @returns {Cohort}
  */
-export async function createCohort() {
+export async function createCohort(cohortName) {
+  if (typeof cohortName !== 'string') {
```

Comment:
The next stage of this is to read through https://www.prisma.io/docs/concepts/components/prisma-client/custom-validation

And use a validation library (prisma doesn't feature it, but supports 3rd party libraries)